mobile/lib/app.dart

```diff
@@ -69,6 +72,11 @@ class _EnteAppState extends State<EnteApp> with WidgetsBindingObserver {
         await MemoryHomeWidgetService.instance.memoryChanged();
       },
     );
+    _peopleChangedSubscription = Bus.instance.on<PeopleChangedEvent>().listen(
+      (event) async {
+        await PeopleHomeWidgetService.instance.peopleChanged();
```

Comment:
What's the reason for refreshing this on PeopleChange? Also this event might be called multiple times during certain syncs, we should ideally have some debouncer or deferred update logic.

Reply:
I was thinking of making this more specific since PeopleChangedEvent can be called even if non-selected people have changed. Debouncer is also good way.